Applications/Games/Origin/Local (Legacy)/script.js

```diff
@@ -19,18 +19,18 @@ new LocalInstallerScript()
         );
     })
     .postInstall(function (wine, wizard) {
-        var originDir = wine.prefixDirectory() + "drive_c/" + wine.programFiles() + "/Origin/";
+        const originDir = wine.prefixDirectory() + "drive_c/" + wine.programFiles() + "/Origin/";
```

Comment:
This is already done in master: https://github.com/PhoenicisOrg/scripts/blob/347375c2e90007a7caf8a3ccf34294212677a714/Applications/Games/Origin/Local%20(Legacy)/script.js#L22

Reply:
ok